docs/features/spelling-and-grammar-checking.md

```diff
@@ -67,13 +75,18 @@ import WProofreader from '@webspellchecker/wproofreader-ckeditor5/src/wproofread
 ClassicEditor
 	.create( editorElement, {
 		plugins: [ ..., WProofreader ],
+		toolbar: [ ..., 'wproofreader' ]
 		wproofreader: {
 			serviceId: 'your-service-ID',
 			srcUrl: 'https://svc.webspellchecker.net/spellcheck31/wscbundle/wscbundle.js'
 		}
 	} )
 ```
 
+<info-box info>
```

Comment:
I'm not sure about the position of this infobox. This section talks about configuring WProofreader Cloud, which is not related to the toolbar items. Additionally, if someone will go directly to WProofreader Server configuration, this infobox won't be visible. I think it would make more sense to add toolbar configuration to the [main installation](https://github.com/ckeditor/ckeditor5/blob/master/docs/features/spelling-and-grammar-checking.md#installation) snippet and move this infobox there. 

Also, do we want to duplicate this infobox? It's also visible above the live demo - won't it be enough to keep it in the installation section which talks about the configuration and available options? WDYT?

Reply:
I merged these two infoboxes to retain the toolbar config information and link.